webapp/package-lock.json

```diff
@@ -2625,7 +2625,7 @@
     "amdefine": {
       "version": "1.0.1",
       "resolved": "https://registry.npmjs.org/amdefine/-/amdefine-1.0.1.tgz",
-      "integrity": "sha1-SlKCrBZHKek2Gbz9OtFR+BfOkfU=",
```

Comment:
I wonder how all these changes were introduced

Reply:
It's been automatically added after install and local deploy. I can remove them from the PR but I'm afraid they'll come again and again in future PRs.